frontend/coprs_frontend/coprs/logic/complex_logic.py

```diff
@@ -133,6 +175,8 @@ def fork_copr(cls, copr, user, dstname, dstgroup=None):
         created = (not bool(forking.get(copr, dstname)))
         fcopr = forking.fork_copr(copr, dstname)
 
+        cls._wait_for_copr_to_be_deleted(fcopr.owner_name, fcopr.name)
```

Comment:
I don't think we can delay the user's API request here :-( this would hang e.g. the `copr-cli` command for 60 seconds at least.  We should accept the request, and delay its processing.